geemap/conversion.py

```diff
@@ -913,20 +911,19 @@ def get_js_examples(out_dir=None):
     Returns:
         str: The folder containing the JavaScript examples.
     """
-    pkg_dir = os.path.dirname(pkg_resources.resource_filename("geemap", "geemap.py"))
-    example_dir = os.path.join(pkg_dir, "data")
-    js_dir = os.path.join(example_dir, "javascripts")
+    pkg_dir = Path(__file__).parent
+    example_dir = pkg_dir / "data"
+    js_dir = example_dir / "javascripts"
 
-    files = list(Path(js_dir).rglob("*.js"))
+    files = list(js_dir.rglob("*.js"))
     if out_dir is None:
         out_dir = js_dir
     else:
-        if not os.path.exists(out_dir):
-            os.makedirs(out_dir)
+        if not out_dir.exists():
+            out_dir.mkdir(parent=True)
```

Comment:
This can actually be replaced by a 1 liner

```suggestion
        out_dir.mkdir(parent=True, exist_ok=True)
```